modules/src/main/java/eu/itesla_project/modules/online/OnlineWorkflowParameters.java

```diff
@@ -114,6 +117,7 @@ public OnlineWorkflowParameters(DateTime baseCaseDate, int states, Interval hist
         this.limitReduction = limitReduction;
         this.handleViolationsInN = handleViolationsInN;
         this.constraintMargin = constraintMargin;
+        this.caseFile = caseFile;
```

Comment:
it's true that it's not enforced, but the command help explains how's working :
"case file - Note:  override (base-case, case-type, countries) parameters"
Anyway, I'll add a more "foolproof" check, if you deem it necessary ...